src/components/text/text.api.json

```diff
@@ -12,16 +12,182 @@
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Guidelines",
+        "sections": [
+          {
+            "type": "section",
+            "title": "Section",
+            "description": "Examples of using the Text component",
+            "content": [
+              "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-veryWide.png?raw=true"
+            ]
+          },
+          {
+            "type": "section",
+            "layout": "horizontal",
+            "title": "Horizontal Section",
+            "description": "Examples of using the Text component",
+            "content": [
+              "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true",
+              "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true"
+            ]
+          },
+          {
+            "type": "list",
+            "title": "List",
+            "description": "Examples of using the Text component",
+            "content": [
+              {
+                "title": "Item #1",
+                "description": "This is item #1",
+                "content": [
+                  "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-veryWide.png?raw=true"
+                ]
+              },
+              {
+                "title": "Item #2",
+                "description": "This is item #2",
+                "content": [
+                  "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-veryWide.png?raw=true"
+                ]
+              }
+            ]
+          },
+          {
+            "type": "list",
+            "layout": "horizontal",
+            "title": "List Horizontal Items",
+            "description": "Examples of using the Text component",
+            "content": [
+              {
+                "title": "Item #1",
+                "description": "This is item #1",
+                "content": [
+                  "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true"
+                ]
+              },
+              {
+                "title": "Item #2",
+                "description": "This is item #2",
+                "content": [
+                  "https://github.com/wix/react-native-ui-lib/blob/master/demo/showcase/Docs/thumbnail-horizontal.png?raw=true"
+                ]
+              }
+            ]
+          }
+        ]
+      },
+      {
+        "title": "Code",
+        "sections": [
+          {
+            "title": "Usage",
+            "type": "usage"
+          },
+          {
+            "type": "info"
+          },
+          {
+            "type": "tip"
+          },
```

Comment:
In the case of these sections (info and tip) It's better if we'll move the content here as well instead of taking it from the component basic info
This will allow us to add multiple tips/infos in different areas in the component's page